src/devices/adurosmart.ts

```diff
@@ -130,9 +130,9 @@ const definitions: Definition[] = [
         meta: {applyRedFix: true},
     },
     {
-        zigbeeModel: ['AD-81812'],
+        zigbeeModel: ['AD-ColorTemperature3001'],
```

Comment:
```suggestion
        zigbeeModel: ['AD-81812', 'AD-ColorTemperature3001'],
```

The old one should stay, otherwise people suddenly have unsupported devices 😄

Reply:
Sorry, I am not the expert in this; Should I keep the old one and just append the "fixed" version or is there another way to have two model/vendor names neatly be linked together.

In either case, considering I added the `'AD-81812` converter in the first place, I have my doubts about whether it will affect anyone else. 

Thanks for your time.